source/get-started/bootable-usb.rst

```diff
@@ -88,11 +92,11 @@ Burn the |CL| image onto a USB drive
 
       sudo umount /dev/sdd*
 
-#. Burn the image onto the USB drive. The example below burns an image onto `<your USB device>`:
+#. Burn the image onto the USB drive. This example burns an image onto `/dev/sdd`.  The parent drive number of the usb varies. 
```

Comment:
I don't understand what "The parent drive number of the usb varies." is trying to say.

Reply:
@mvincerx - we should "spell out" the example USB device for the MacOS portion too.  

Current:
sudo dd if=./clear-[version number]-live-[desktop | server].iso of=**<your USB device>** bs=4m

Change:
sudo dd if=./clear-[version number]-live-[desktop | server].iso of=/dev/disk2 bs=4m

Reply:
I will replace that sentence (The parent drive..) with `The device name will vary. `

Will follow through with changes to Mac section. Good call.

Reply:
@bktan8 See latest commit.